### PR TITLE
zh-translation: about/contribute/shortcodes/index.md

### PR DESCRIPTION
**Pull request by Ruil1n**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/about/contribute/shortcodes/index.md [hunk 1]
```diff
@@ -116,22 +116,22 @@ Hugo 的 shortcode 是具有特定语法的特殊占位符，您可以将其添
 当您在页面中介绍一个 Istio 术语时，贡献补充条款要求您将该术语包含在 `glossary` 中，并使用 shortcode `{{</* gloss */>}}` 标记它的第一个实例。shortcode 会对其进行特殊渲染，读者点击该术语，可以在弹出的窗口中获取该术语的定义。例如：
 
 {{< text markdown >}}
-Mixer 使用 {{</*gloss*/>}}adapters{{</*/gloss*/>}} 与后端进行交互。
+Istio 里的组件可以控制 {{</*gloss*/>}}Envoy{{</*/gloss*/>}} 代理，负责服务发现、负载均衡和路由分发。
```
Comment (kebe7jun) [inline content/zh/about/contribute/shortcodes/index.md:119]:
```suggestion
Istio 里的组件通过控制 {{</*gloss*/>}}Envoy{{</*/gloss*/>}} 代理，实现服务发现、负载均衡和路由分发。
```

Comment (kebe7jun) [reply content/zh/about/contribute/shortcodes/index.md:119]: 后面的几个也改一下。

Comment (Ruil1n) [reply content/zh/about/contribute/shortcodes/index.md]: 已修正

**Review by kebe7jun — APPROVED** [review]
LGTM

**Pull request merged**